tests/features/capsulahub-application.feature

```diff
@@ -0,0 +1,80 @@
+
+ Feature: CapsulaHub application
+
+ Background:
+    Given a npm package called @capsulajs/capsula-hub
+    And   a default port 55555
+    And   a valid port is a number between 1024 and 65535
+    And   two existing config files conf.js and conf1.js located in capsulahub/src/config
+
+ Scenario: Run Capsula-Hub  instance with a valid config file path without specifying the port
+    When  I run the command `capsula-hub run -l <path>` in the relevant directory
+		 | <path>         |
+		 | conf.js        |
+		 | config/conf.js |
+		 | src/config/conf.js|
+    Then  the application is running on the default port
+    And   workspace is created with the specified config file
```

Comment:
As I understand, we will have to go to that localhost to check what has been rendered in browser to check that Workspace has been created with an appropriate config.